.github/build-scripts/patch-wheel.sh

```diff
@@ -43,6 +41,23 @@ to_rewrite=$(ldd libuct_cuda.so | awk '/libcudart/ { print $1 }')
 patchelf --replace-needed ${to_rewrite} libcudart-${hash}.so libuct_cuda.so
 patchelf --add-rpath '$ORIGIN' libuct_cuda.so
 
+find /lib/x86_64-linux-gnu/ -name "libnvidia-ml*.so.*" | xargs cp -P -t .
+src=libnvidia-ml.so.1
```

Comment:
Here, it looks like `libnvidia-ml.so` also exists, so maybe start there (and adapt the previous line to `libnvidia-ml*.so*` ?